bundles/com.espressif.idf.ui/OSGI-INF/l10n/bundle.properties

```diff
@@ -22,4 +22,6 @@ editor.name = ESP-IDF Size Analysis
 page.name = Espressif
 
 wizard.name.1 = Espressif IDF Component
-wizard.description.0 = Create a new EspressIf IDF component
\ No newline at end of file
+wizard.description.0 = Create a new EspressIf IDF component
+command.label.2 = Python clean
```

Comment:
Can we change it to upper case - "Python Clean"

Reply:
@kolipakakondal changed to ESP-IDF: Python Clean
<img width="331" alt="Screenshot 2021-06-23 at 15 55 55" src="https://user-images.githubusercontent.com/24419842/123109461-8c7b9380-d43b-11eb-83d5-65bd65c7d60d.png">